demo/main.js

```diff
@@ -235,6 +235,9 @@ $(document).ready(function () {
       toggleTab($('.demo-tab-btn')[parseInt(indexString) || 0], true);
     });
   }
+  $(window).on('popstate', function () {
+    location.reload();
```

Comment:
I think it would be better to access it on `window`, like you did with `location` later :)

```suggestion
    window.location.reload();
```